app/src/main/kotlin/at/bitfire/davdroid/ui/intro/PermissionsIntroPage.kt

```diff
@@ -31,4 +31,9 @@ class PermissionsIntroPage: IntroPage {
         PermissionsContent()
     }
 
+    // Check whether permissions have changed after user comes back from settings app
+    override fun onResume(application: Application) {
+        PermissionsActivity.Model(application).checkPermissions()
```

Comment:
This line creates a new Model (and doesn't re-use the model that is created in `PermissionsContent()`).

Reply:
Ah, of course. I was a tad too quick here it seems ...